indigo_app/templates/indigo_app/place/_work_chooser.html

```diff
@@ -0,0 +1,85 @@
+{% load i18n %}
+
+<div class="modal-dialog modal-xl" data-component="WorkChooser">
+  <div class="modal-content">
+    <div class="modal-header">
+      <h5 class="modal-title">{% trans 'Choose a work' %}</h5>
+      <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+    </div>
+
+    <div class="card">
+      <div class="card-header">
+        <form
+          id="work-chooser-filter-form"
+          hx-get="{% url 'place_work_chooser_list' place.place_code %}"
+          hx-trigger="keyup changed from:input delay:0.5s, submit"
+          hx-target=".work-list-table tbody"
+        >
+          <input type="hidden" name="field" value="{{ work_field }}">
+          <input type="hidden" name="submit" value="{{ hx_submit }}">
+          <input type="hidden" name="target" value="{{ hx_target }}">
+          <div class="row">
+
+            <div class="col-3">
+              <select
```

Comment:
When the country or locality changes, we update the entire form so that we can update the dropdowns, too.